tests/images/customize.sh

```diff
@@ -145,45 +145,118 @@ qemu_disk() {
 }
 
 assert() {
-    grep -Fq "$1" log
+    if grep -Fq "$1" log; then
+        echo "Assertion passed: '$1' found in log"
+        return 0
+    else
+        echo "Assertion failed: '$1' not found in log"
+        return 1
+    fi
+}
+
+assert() {
+   grep -Fq "$1" log
+}
+
+assert_not() {
+    if grep -Fq "$1" log; then
+        return 1
+    else
+        return 0
```

Comment:
Should include echo statements here too for consistency?

Reply:
Sorry I thought I had removed the first assert() it was too much as the asserts now echo.